bugbug/tools/code_review.py

```diff
@@ -129,26 +286,87 @@ def get_review_request_by_id(self, revision_id: int) -> ReviewRequest:
         assert len(revisions) == 1
         return ReviewRequest(revisions[0]["fields"]["diffID"])
 
+    @tenacity.retry(
+        stop=tenacity.stop_after_attempt(7),
+        wait=tenacity.wait_exponential(multiplier=1, min=16, max=64),
+        reraise=True,
+    )
     def get_patch_by_id(self, patch_id: int) -> Patch:
         assert phabricator.PHABRICATOR_API is not None
         raw_diff = phabricator.PHABRICATOR_API.load_raw_diff(int(patch_id))
         return Patch(raw_diff)
 
+    def _get_revisions_count(self):
+        return sum(1 for _ in phabricator.get_revisions())
+
+    def get_all_inline_comments(
+        self, comment_filter
+    ) -> Iterable[tuple[int, list[InlineComment]]]:
+        db.download(phabricator.REVISIONS_DB)
+
+        for revision in tqdm(
+            phabricator.get_revisions(), total=self._get_revisions_count()
```

Comment:
Given it's only used here and internal, you could just inline this function or have a variable:
```Python
        revision_count = sum(1 for _ in phabricator.get_revisions())
        for revision in tqdm(phabricator.get_revisions(), total=revision_count)
```
or
```Python
        for revision in tqdm(phabricator.get_revisions(), total=sum(1 for _ in phabricator.get_revisions()))
```

Given iterating through revisions is expensive, you could also just get rid of `total` and use an indeterminate progress bar:
```Python
        for revision in tqdm(phabricator.get_revisions())
```

Reply:
I implemented the count using a separate method so we could use a cache decorator if we need to use it somewhere else later. Since we do not currently use it somewhere else, maybe we do not need it for now. 

Done in 15c2ab54f2af83528658b591a4de10418dd58e2e